workflows/python/sdk/README.md

```diff
@@ -83,21 +112,16 @@ For a more detailed view of the workflow activities (duration, progress etc.), t
 
 ### What happened? 
 
-When you ran `dapr run --app-id order-processor --resources-path ../../../components/ -- python3 app.py`
-
-1. First the user inputs an order for 10 cars into the concole app.
-2. A unique order ID for the workflow is generated (in the above example, `b903d749cd814e099f06ebf4a56a2f90`) and the workflow is scheduled.
-3. The `NotifyActivity` workflow activity sends a notification saying an order for 10 cars has been received.
-4. The `VerifyInventoryActivity` workflow activity checks the inventory data, determines if you can supply the ordered item, and responds with the number of cars in stock.
-5. The `RequestApprovalActivity` workflow activity is triggered due to buisness logic for orders exceeding $50k and user is prompted to manually approve the purchase before continuing order. 
-6. The workflow starts and notifies you of its status.
-7. The `ProcessPaymentActivity` workflow activity begins processing payment for order `b903d749cd814e099f06ebf4a56a2f90` and confirms if successful.
-8. The `UpdateInventoryActivity` workflow activity updates the inventory with the current available cars after the order has been processed.
-9. The `NotifyActivity` workflow activity sends a notification saying that order `b903d749cd814e099f06ebf4a56a2f90` has completed.
-10. The workflow terminates as completed.
-
-
-
-
+When you ran `dapr run -f .`
 
+1. An OrderPayload is made containing one car.
+2. A unique order ID for the workflow is generated (in the above example, `fc8a507e4a2246d2917d3ad4e3111240`) and the workflow is scheduled.
+3. The `notify_activity` workflow activity sends a notification saying an order for one car has been received.
+4. The `verify_inventory_activity` workflow activity checks the inventory data, determines if you can supply the ordered item, and responds with the number of cars in stock. The inventory is sufficient so the workflow continues.
+5. The total cost of the order is 5000, so the workflow will not call the `request_approval_activity` activity.
+6. The `process_payment_activity` workflow activity begins processing payment for order `fc8a507e4a2246d2917d3ad4e3111240` and confirms if successful.
+7. The `update_inventory_activity` workflow activity updates the inventory with the current available cars after the order has been processed.
+8. The `notify_activity` workflow activity sends a notification saying that order `fc8a507e4a2246d2917d3ad4e3111240` has completed.
+9. The workflow terminates as completed and the OrderResult is set to processed.
 
+> **Note:** This quickstart uses an OrderPayload of one car with a total cost of $5000. Since the total order cost is not over 5000, the workflow will not call the `RequestApprovalActivity` activity nor wait for an approval event. Since the quickstart is a console application, it can't accept incoming events easily. If you want to test this scenario, convert the console app to a service and use the `DaprWorkflowClient` to send an event to the workflow.
```

Comment:
Does it make sense to have `DaprWorkflowClient` for each language? Feel like its different format for each so might make sense to change this to just Dapr workflow client and link: https://v1-15.docs.dapr.io/reference/api/workflow_api/#raise-event-request